corehq/apps/reports/views.py

```diff
@@ -1856,6 +1856,7 @@ class TableauServerView(BaseProjectReportSectionView):
     page_title = gettext_lazy('Tableau Server Config')
     template_name = 'hqwebapp/crispy/single_crispy_form.html'
 
+    @cls_require_superusers
```

Comment:
Is there a reason for using this instead of `@method_decorator(require_superuser)`?